src/s390x/zipl.rs

```diff
@@ -28,109 +27,54 @@ use walkdir::WalkDir;
 /// # Arguments
 /// * `boot` - path to boot partition mountpoint, i.e. smth like /boot
 /// * `disk` - smth like /dev/dasda
+/// * `firstboot_kargs` - enables ignition firstboot and adds provided kargs
 pub fn install_bootloader<P: AsRef<Path>>(
     boot: P,
-    disk: &str,
-    extra_kargs: Option<&str>,
+    disk: Option<&str>,
```

Comment:
I'm confused by this; why is this an `Option` now?

Reply:
i want to make this function re-usable by `rdcore`, in this case we don't need to re-ipl machine again, so `disk` is an Option

Reply:
Ack, gotcha. Maybe we should change the docstring to something like:

> /// * `disk` - optional device path on which to run `chreipl`

?